easybuild/tools/version.py

```diff
@@ -43,7 +43,7 @@
 # recent setuptools versions will *TRANSFORM* something like 'X.Y.Zdev' into 'X.Y.Z.dev0', with a warning like
 #   UserWarning: Normalizing '2.4.0dev' to '2.4.0.dev0'
 # This causes problems further up the dependency chain...
-VERSION = LooseVersion('4.5.1.dev0')
+VERSION = LooseVersion('4.5.2.dev')
```

Comment:
```suggestion
VERSION = LooseVersion('4.5.2.dev0')
```

If I've understood the comment correctly and the previous version here.

Reply:
Fixed with force push in 015493f, since this was actually causing trouble...

```
$ eb --install-latest-eb-release
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/kehoste/work/easybuild-framework/easybuild/main.py", line 70, in <module>
    from easybuild.tools.options import set_up_configuration, use_color
  File "/Users/kehoste/work/easybuild-framework/easybuild/tools/options.py", line 203, in <module>
    class EasyBuildOptions(GeneralOption):
  File "/Users/kehoste/work/easybuild-framework/easybuild/tools/options.py", line 205, in EasyBuildOptions
    VERSION = this_is_easybuild()
  File "/Users/kehoste/work/easybuild-framework/easybuild/tools/version.py", line 93, in this_is_easybuild
    top_version = max(FRAMEWORK_VERSION, EASYBLOCKS_VERSION)
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/version.py", line 64, in __gt__
    c = self._cmp(other)
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/version.py", line 341, in _cmp
    if self.version < other.version:
TypeError: '<' not supported between instances of 'int' and 'str'
```